core/src/main/resources/reference.conf

```diff
@@ -93,6 +93,10 @@ akka.kafka.consumer {
   # This value is used instead of Kafka's default from `default.api.timeout.ms`
   # which is 1 minute.
   metadata-request-timeout = 5s
+
+  # Interval for checking that transaction was completed before closing the consumer.
+  # Used in the transactional flow.
+  draining-check-interval = 30ms
```

Comment:
I wonder if the setting name should contain `eos` or something. There are so many settings now it is hard to see which belong together.

Reply:
tbh... `eos` is cryptic even for me...

Reply:
but we may have some distinct section for transactional processing?